src/main/java/mate/academy/service/AuthenticationService.java

```diff
@@ -10,7 +12,10 @@ public class AuthenticationService {
      * @return true if user by email exists and passed password is equal to user's password.
      * Return false in any other cases.
      */
+
     public boolean login(String email, String password) {
-        return false;
+        UserService userService = new UserService();
```

Comment:
in previous comment, I mentioned that UserService should be class-level variable
Now, you create new instanse of UserService each time when you call login method